Dockerfile

```diff
@@ -1,20 +1,22 @@
 FROM ubuntu
-MAINTAINER CaffeineAddiction
 
-RUN apt update && \
-    apt install build-essential python2.7 python-dev git wget -y && \
-    wget https://bootstrap.pypa.io/get-pip.py && \
-    python2.7 get-pip.py && \
-    cd ~ && \
-    env GIT_SSL_NO_VERIFY=true git clone https://github.com/Charcoal-SE/SmokeDetector.git && \
-    cd SmokeDetector && \
-    git submodule init && \
-    git submodule update && \
-    pip install -r requirements.txt --upgrade
-
-RUN apt remove -y --purge build-essential python-dev wget && \
+RUN DEBIAN_FRONTEND=noninteractive && \
+    apt -y update && \
+    apt install -y build-essential \
+        python3 python3-dev python3-pip python3-venv git && \
+    adduser --disabled-password --force-badname SmokeDetector \
+        --gecos SmokeDetector && \
+    su - SmokeDetector sh -c '\
+        env GIT_SSL_NO_VERIFY=true git clone https://github.com/Charcoal-SE/SmokeDetector.git && \
```

Comment:
Why `GIT_NO_SSL_VERIFY`?

Reply:
I just copy/pasted it from the old `Dockerfile` but you get weird errors from `git` if you don't have the full certificate chain for SSL verification. I'd imagine installing the `ca-certificates` package would fix that more properly. Thanks for inquiring; I'll see if I can get rid of that.

Reply:
@tripleee Any updates on that?  Because I'd rather use valid SSL, not invalid SSL.  (`ca-certificates` is important after all to have so Smokey when it runs doesn't blow up on bad SSL)